deploy/infra/security-addons.bicep

```diff
@@ -0,0 +1,106 @@
+targetScope='subscription'
```

Comment:
Is this template Idempotent?

Reply:
Need to create all private endpoints in the separate subnet that are not created by main.bicep. I observed install.sh failed when I rerun the same command with a complaining that it can not delete existing private endpoints. 
It means that since main.bicep does not know private endpoints created by security-addons.bicep, it tries to delete private endpoints when rerunning. So, I will take a look at how we resolve this issue.